packages/fxa-settings/Gruntfile.js

```diff
@@ -37,6 +37,23 @@ module.exports = function (grunt) {
         dest: 'test/settings.ftl',
       },
     },
+    hash: {
+      options: {
+        mapping: 'public/static/static-asset-manifest.json', // The file where the hashed file names will be stored
+        srcBasePath: 'public/', // the base Path you want to remove from the `key` string in the mapping file
+        destBasePath: 'public/static',
+      },
+      locales: {
+        expand: true,
+        cwd: 'public/locales',
+        src: '**/*.ftl', //all your js that needs a hash appended to it
```

Comment:
Nit, should these comments say "we" instead of "you" and FTL instead of JS?

Reply:
Ah, this is just copy pasta from their docs example. Let me fix this...